src/nisarqa/utils/stats_h5_writer/setup_writer.py

```diff
@@ -43,6 +47,57 @@ def setup_stats_h5_all_products(
     log.info(f"Input file's runconfig copied to {stats_file}")
 
 
+def add_global_metadata_to_stats_h5(
+    product_type: str, stats_h5: h5py.File
+) -> None:
+    """
+    Write global metadata as Attributes to the root directory of a HDF5 file.
+
+    Parameters
+    ----------
+    product_type : str
+        One of: 'rslc', 'gslc', 'gcov', 'rifg', 'runw', 'gunw', 'roff', 'goff'.
+    stats_h5 : h5py.File
+        Handle to an h5 file where the global metadata will be written.
+
+    See Also
+    --------
+    nisarqa.utils.plotting.add_metadata_to_report_pdf
+        Sister function which adds global metadata to the REPORT.pdf.
+    """
+
+    if product_type.lower() not in nisarqa.LIST_OF_NISAR_PRODUCTS:
+        raise ValueError(
+            f"{product_type=}, must one of: {nisarqa.LIST_OF_NISAR_PRODUCTS}"
+        )
+
+    product_type = product_type.upper()
+
+    global_attrs = {
+        "Conventions": "CF-1.7",
```

Comment:
Let's remove for now since there's no known requirement for QA products to be CF-compliant, we haven't made any attempt to make them CF-compliant, and there's no science quality data contained within that end users will need to use with CF-compatible software.

```suggestion
```